tests/libclient/libclient.go

```diff
@@ -27,8 +27,11 @@ import (
 
 	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	v1 "kubevirt.io/api/core/v1"
+	"kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1"
```

Comment:
Please keep the cdi naming through an alias: `cdiv1beta1`

Reply:
DONE